packages/vscode-plugin/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
```

Comment:
This looks good. We'll just have to remember to swap this out when bump the version. I might write a script for this.